ldm/invoke/_version.py

```diff
@@ -1 +1 @@
-__version__='2.3.1.post2'
+__version__='2.3.2.dev0'
```

Comment:
are you sure .dev0 will work with pep - shouldn't it be `2.3.2-a0` 🤔

Reply:
I dunno. All these PEPs confuse me. Here's what PEP-440 says:

```
Public version identifiers are separated into up to five segments:

Epoch segment: N!
Release segment: N(.N)*
Pre-release segment: {a|b|rc}N
Post-release segment: .postN
Development release segment: .devN
```

Reply:
In any case, I'm bumping to 2.3.2.